src/main/resources/xml/schema/2020-1/nordic-html5.rng

```diff
@@ -231,6 +251,15 @@
         <optional>
             <ref name="attr.xmlspace"/> <!-- @xml:space -->
         </optional>
+        <optional>
+            <ref name="attr.role"/> <!-- @role -->
+        </optional>
+        <optional>
+            <ref name="attr.aria.label"/> <!-- @aria-label -->
+        </optional>
+        <optional>
+            <ref name="attr.aria.labelledby"/> <!-- @aria-labelledby -->
+        </optional>
```

Comment:
I think this will cause problems in instances where either role, aria-label or aria-labelledby are defined explicitly.

For instance, this is not valid RelaxNG:

```xml
<grammar xmlns="http://relaxng.org/ns/structure/1.0">
    <start>
        <element name="test">
            <attribute name="role"/>
            <optional>
                <attribute name="role"/>
            </optional>
        </element>
    </start>
</grammar>
```

It gives me the error `duplicate attribute "role"` in Oxygen XML Editor.

This is why we have the `*.notype` and `*.noclass` definitions, so that we can reference those when a `epub:type` or `class` have been explicitly defined.

See for instance:
https://github.com/nlbdev/nordic-epub3-dtbook-migrator/blob/45007dd2616e82e42028eb012e139879b8cd79e2/src/main/resources/xml/schema/2020-1/nordic-html5.rng#L743-L751

When both a `epub:type` and `class` is defined, we could use `attrsrqd.base` (this one) directly, as it essentially meant "core attributes except epub:type and class". Here's an example of this from the 2015-1 version of this file:
https://github.com/nlbdev/nordic-epub3-dtbook-migrator/blob/8f80992fbfcc24799ce99a071e41a2690611cfef/src/main/resources/xml/schema/nordic-html5.rng#L1633-L1648

So since we now have three new attributes, it gets more complicated. We might need a `coreatts.norole`, `coreatts.nolabel`, `coreatts.nolabelledby` for the cases where those attributes are not defined. Other combinations might even be useful as well, like `coreattrs.norolenoclass` if both a `role` and a `class` attribute are explicitly defined.

Reply:
Hi @josteinaj 

I see your point, and there might be a reason to specify a ```coreatts.norole```  if we want to create a specific role to decide a structure. We don't have that need yet, but we could make that change in the future.

When it comes to ```aria-label``` and ```aria-labelledby```  there is no such need IMHO.

The label can be any text and will not specify a type and the labeled by id is just a pointer.

Best regards
Daniel

Reply:
Good points.

Then maybe `aria-describedby` also should be declared as an attribute that can occur anywhere? If so, we wouldn't need to declare it in the image figure declaration (see discussion in #432).